src/vs/workbench/contrib/search/browser/searchActionsFind.ts

```diff
@@ -335,6 +360,16 @@ function getMultiSelectedSearchResources(viewer: WorkbenchCompressibleObjectTree
 		.filter((renderableMatch): renderableMatch is URI => (renderableMatch !== null));
 }
 
+function expandSelectSubtree(accessor: ServicesAccessor) {
+	const viewService = accessor.get(IViewsService);
+	const searchView = getSearchView(viewService);
+	if (searchView) {
+		const viewer = searchView.getControl();
+		const selected = viewer.getFocus()[0];
+		viewer.expand(selected);
```

Comment:
This only expands one level. The call will need to be such that recursive is true.